### PR TITLE
System.BitConverter F# snippets

### PR DESCRIPTION
**Pull request by albert-du**
Adds F# snippets for BitConverter

### CODE REVIEW

**Review by dsyme — APPROVED** [review]

File: samples/snippets/fsharp/VS_Snippets_CLR_System/system.BitConverter.DoubleInt64/FS/bitstodbl.fs [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+longBitsToDouble 0x41EFFFFFFFE00000L
+longBitsToDouble 0x3F70000000000000L
+longBitsToDouble 0x3DF0000000000000L
+longBitsToDouble 0x0000000000000001L
+longBitsToDouble 0x000000000000FFFFL
+longBitsToDouble 0x0000FFFFFFFFFFFFL
+longBitsToDouble 0xFFFFFFFFFFFFFFFFL       //
+longBitsToDouble 0xFFF0000000000000L            //
+longBitsToDouble 0x7FF0000000000000L
+longBitsToDouble 0xFFEFFFFFFFFFFFFFL        //
```
Comment (BillWagner) [inline samples/snippets/fsharp/VS_Snippets_CLR_System/system.BitConverter.DoubleInt64/FS/bitstodbl.fs:34]: I'm curious about the trailing `//` on these lines. Is there are purpose?

**Comment by BillWagner** [comment]
Overall, this is ready to merge @albert-du 

I had one question, and once we resolve it, I'll :shipit:

**Review by BillWagner — APPROVED** [review]
Thanks again @albert-du 

I'll :shipit:

File: samples/snippets/fsharp/VS_Snippets_CLR_System/system.BitConverter.DoubleInt64/FS/bitstodbl.fs (hunk 1 above)
Comment (albert-du) [reply samples/snippets/fsharp/VS_Snippets_CLR_System/system.BitConverter.DoubleInt64/FS/bitstodbl.fs]: no idea how those got there, I've removed them

**Pull request merged**